gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -61,16 +65,30 @@ public class AndroidCommandPluginExtension {
     // prefer system property over direct setting to enable commandline arguments
     @Memoized
     def getDeviceId() {
-        if (System.properties['deviceId'])
+        if (System.properties['deviceId']) {
             return System.properties['deviceId']
-        if (deviceId instanceof Closure)
+        } else if (deviceId instanceof Closure) {
             return deviceId.call()
+        }
         deviceId ?: defaultDeviceId()
     }
 
-    // prefer system property over direct setting to enable commandline arguments
     def getEvents() {
-        System.properties['events'] ?: events ?: 10000
+        if (System.properties['events']) {
+            return System.properties['events']
+        } else if (events) {
+            return events
+        }
+        EVENTS_DEFAULT
+    }
```

Comment:
Yeah, I get the usage of `?:` operator. 

I am happy to change it back to the nested/double elvis  and bring back the comment as long as somebody would give two :+1:  and merge my changes :) 

I believe it has been already pointed out that it's a drawback of Groovy and all these other new languages that they don't have/we don't use static analysis tools and do this kind of job for us.